Client/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -86,9 +86,34 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
         view.addSubview(blur)
 
         super.viewDidLoad()
+        setupSearchEngingScrollViewIfNeeded()
 
         KeyboardHelper.defaultHelper.addDelegate(self)
 
+        blur.snp.makeConstraints { make in
+            make.edges.equalTo(self.view)
+        }
+
+        suggestionCell.delegate = self
+
+        NotificationCenter.default.addObserver(self, selector: #selector(dynamicFontChanged), name: .DynamicFontChanged, object: nil)
+    }
+
+    @objc func dynamicFontChanged(_ notification: Notification) {
+        guard notification.name == .DynamicFontChanged else { return }
```

Comment:
Can you flip this condition to use if?
I know it was there before your PR, we usually use guard conditions only when at least one item needs unwrapping

Reply:
Sure thing - Usually you would use a `guard` if you want to return early as a precondition without adding an additional . Happy to change to your standards.